app/api/login.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Login < Grape::API
+  format :json
+
+  desc 'Login Endpoint'
+  namespace :login do
+    desc 'Login with email and password'
+    params do
+      requires :email, type: String, desc: 'email'
+      requires :password, type: String, desc: 'password'
+    end
+    post do
+
+      user = User.find_by_email params[:email]
```

Comment:
Up to you, not necessary, yet easier to read, we might consider adding parentheses around the argument, wdyt?